egs/swbd/s5c/local/nnet3/run_lstm.sh

```diff
@@ -34,6 +34,7 @@ non_recurrent_projection_dim=256
 chunk_width=20
 chunk_left_context=40
 chunk_right_context=0
+ivector_interval=10
```

Comment:
OK, what I'm saying more specifically is that you should get the ivector_period from the directory containing the iVectors.  There's an option to specify it in the script that dumps the iVectors, but there is no need to have an option in the run*.sh scripts setting that.
Instead you can just get it from ivector_dir/ivector_period.  You can pass it in as a flag to the make_configs.py script, but the get_egs.sh scripts can obtain it themselves from the ivector_dir/ivector_period file, since they get the ivector_dir as an argument.